dockers/docker-orchagent/supervisord.conf

```diff
@@ -70,6 +70,15 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=orchagent:running
 
+[program:gearsyncd]
+command=/usr/bin/gearsyncd -p /usr/share/sonic/hwsku/gearbox_config.json
+priority=4
```

Comment:
why the priority set to 4 ? It's same as orchagent.

Reply:
We could put it at 3 same as portsyncd. Would that be ok for you?

Reply:
The priority value can be the same as another program. But, we normally set the priorities to be in the order with which we would like the programs to start. However, Supervisor does not have a built in dependency mechanism, so the priority doesn't help prevent programs from starting simultaneously. This is why we've added the supervisor-dependent-startup add-on package. But for the sake of being consistent with the other programs in the file and , I suggest moving this section into a position in the file where you would like it started (it helps give a visual representation), and you should probably also add the following lines:

```
dependent_startup=true
dependent_startup_wait_for=swssconfig:exited
```